source/Calamari.Aws/Deployment/Conventions/CreateCloudFormationChangeSetConvention.cs

```diff
@@ -41,10 +41,7 @@ string iamCapabilities
             this.stackProvider = stackProvider;
             this.roleArnProvider = roleArnProvider;
             this.templateFactory = templateFactory;
-            if (iamCapabilities.IsKnownIamCapability())
-            {
-                Capabilities.Add(iamCapabilities);
-            }
+            Capabilities.AddRange(iamCapabilities.Where(x => x.IsKnownIamCapability()));
```

Comment:
Should we add a warning or something, to at least let the user know they passed something invalid?